package.json

```diff
@@ -23,7 +23,7 @@
     "camelcase-keys": "^6.1.2",
     "core-js": "^3.6.4",
     "ideogram": "1.19.0",
-    "igv": "2.1.0",
+    "@single-cell-portal/igv": "2.5.6-beta.2",
```

Comment:
nice, yeah, I'm definitely glad you took the effort to make the repo rather than manually including the file